index.html

```diff
@@ -6,6 +6,22 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <link rel="icon" href="/favicon.ico" />
     <title>Cuttle</title>
+
+    
+    <meta property="og:url" content="https://www.cuttle.cards/" />
+    <meta property="og:title" content="Cuttle Cards Online" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="/public/img/cuttle_meta_img.jpg/" />
+    <meta property="og:description"
+        content="Cuttle is a 2 player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free!" />
+
+    <meta name="twitter:site" content="https://unacog.com/">
+    <meta name="twitter:title" content="Unacog About">
+    <meta name="twitter:description"
+        content="Cuttle is a 2 player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free!">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:image:src" content="/public/img/cuttle_meta_img.jpg/">
```

Comment:
Name should use hyphens imo and be named more clearly. Something like `cuttle-twitter-banner`, promo image, etc.

Reply:
yeah sorry - some sloppiness from my project :) please make any necessary adjustments.

Reply:
@seriouslysean you mean for the image filename? If it's the same img file for the `og` and `twitter` protocols, maybe we can stick with a generic name (which @Mal402 we do generally use kebab-case capitalization for):

`cuttle-summary-banner.jpg` maybe?